main.cpp

```diff
@@ -275,296 +300,281 @@ struct Bakery
         bool burn(int bakingTime);
     };
 
-    //bake bread
-    void bakeBread(double flourAmount, RyeBread brd);
-    //sell cakes
+    void bakeBread(double bFlourAmount, RyeBread brd);
```

Comment:
Review our Slack discussion of using member variables in member functions directly. 
Change your argument list so that your function arguments are not duplicates of your member variables.
This can mean changing the function name (meaning the function purpose has changed) 
or it could mean changing the function arguments.
Remember, name things what they are or what they do.
- [ ] revise